src/Microsoft.ML.TimeSeries/SsaAnomalyDetectionBase.cs

```diff
@@ -13,23 +13,23 @@
 
 namespace Microsoft.ML.TimeSeriesProcessing
 {
+    public enum ErrorFunction : byte
+    {
+        SignedDifference,
+        AbsoluteDifference,
+        SignedProportion,
```

Comment:
Those functions need reference links, aspecially, for SignedProportion and AbsoluteProportion.

Reply:
Please refer to previous comment. We plan to upload timeseries specific docs that will contain the overall architecture of time series.

---
In reply to: [254488733](https://github.com/dotnet/machinelearning/pull/2344#discussion_r254488733) [](ancestors = 254488733)